File: Java/src/main/java/MDC.java
public class MDC {

  public void mdc(int a, int b) {
    int aux;
    while (a % b > 0) {
      aux = a % b;
      a = b;
      b = aux;
    }
    System.out.println("Alg_MDC: " + b);
  };

  public MDC(int a, int b) {
    mdc(a, b);
  };
}

File: Trabalhos/Ambientes desenv de software/Entregável #1 - Algoritmos/MDC.js
var a = parseInt(prompt("Insira o valor de A"));
var b = parseInt(prompt("Insira o valor de B"));

function mdc(a, b) {
  var aux;
  while (a % b > 0) {
    aux = a % b;
    a = b;
    b = aux;
  }
  return b;
}

console.log(mdc(a, b));
